VMEncryption/test/test_check_util.py

```diff
@@ -69,41 +69,41 @@ def test_validate_volume_type(self):
     def test_fatal_checks(self):
```

Comment:
Unit tests seem to be missing the case where "OS" was already encrypted, or cases with with less than, equal to, or greater than the required amount of RAM.